modules/tinymce/src/core/main/ts/api/UndoManager.ts

```diff
@@ -50,6 +51,29 @@ const UndoManager = (editor: Editor): UndoManager => {
       return Rtc.addUndoLevel(editor, undoManager, index, locks, beforeBookmark, level, event);
     },
 
+    /**
+     * Dispatch a change event with current editor status as level and current undoManager layer as lastLevel
+     *
+     * @method dispatchChange
+     */
+    dispatchChange: (): void => {
+      const data = undoManager.data;
+      const currentLevel = Levels.createFromEditor(editor);
+
+      const dispatchEvent = (lastLevel: UndoLevel) => {
```

Comment:
This type is incorrect, as you're literally passing `undefined` below.

Reply:
fixed